src/ray/core_worker/transport/actor_scheduling_queue.cc

```diff
@@ -39,17 +39,17 @@ ActorSchedulingQueue::ActorSchedulingQueue(
       ss << "\t" << concurrency_group.name << " : " << concurrency_group.max_concurrency;
     }
     RAY_LOG(INFO) << ss.str();
-    fiber_state_manager_ = std::make_unique<ConcurrencyGroupManager<FiberState>>(
```

Comment:
I think a simple fix is instead of creating a fiber_state_manager per ActorSchedulingQueue, we should have a global one and pass in to the ActorSchedulingQueue constructor. Similar to how we do it for `std::shared_ptr<ConcurrencyGroupManager<BoundedExecutor>> pool_manager,` ?

Reply:
Actually we can combine the two trick. Now we have 1 thread per actor per caller worker per concurrency group.

With removing `ConcurrencyGroupManager` and putting multiple rate limiters in FiberState we can remove "per concurrency group",
With moving it to `CoreWorkerDirectTaskReceiver` we can remove "per caller".

Adding code...

Reply:
The simple fix I proposed is 1 fiber thread per actor per concurrency group. This matches the 1 event loop thread per actor per concurrency group.

I feel the fix can be just creating a `fiber_state_manger_` in `direct_actor_transport.cc` and pass it to all the actor queues similar to how it creates `pool_manager_` and pass to all the actor queues. Am I missing anything here.

Reply:
updated to use per-cg threads.